src/time.cr

```diff
@@ -398,6 +398,11 @@ struct Time
     days[month]
   end
 
+  # Returns how many days are there in a year
+  def self.days_in_year(year : Int) : Int32
+    days_in_month(year, 2) + 337
```

Comment:
A simpler implementation and more explicit: `leap_year?(year) ? 366 : 365`

Reply:
I agree and also faster. I will modify the code for that.